holoviews/plotting/bokeh/raster.py

```diff
@@ -181,6 +186,14 @@ def get_data(self, element, ranges, style):
         img = np.dstack([element.dimension_values(d, flat=False)
                          for d in element.vdims])
 
+        if self.clim_percentile:
+            if isinstance(self.clim_percentile, bool):
+                low, high = np.percentile(img, (2, 98))
+            else:  # True
+                low, high = np.percentile(img, (self.clim_percentile, 100 - self.clim_percentile))
+            img = np.clip(img, low, high)
+            img = img / high * 255
```

Comment:
```suggestion
            img = (img - low) / (high - low) * 255
```

Another thought as I don't think we are using the full range of the RGB.